views/book-no-edit.js

```diff
@@ -0,0 +1,24 @@
+//var used for global scope availability
+var app = app || {};
+// noinspection JSVoidFunctionReturnValueUsed
+app.BookItem = Backbone.View.extend({
+	//<tr> because I am inserting element in a table
+	tagName: 'tr',
+	//when one of the input boxes corresponding to this model is changed update model
+	events: {
+		"change input": 'updateModel',
+	},
+	/**
+	 * Renders the html for a book
+	 * @returns {app.BookItem}
+	 */
+	render: function () {
+		let data = this.model.toJSON();
```

Comment:
you don't need to read use `.toJSON()` just use `this.model.get('title/author/etc')`